src/resqml2_0_1/RockFluidUnitFeature.cpp

```diff
@@ -0,0 +1,36 @@
+/*-----------------------------------------------------------------------
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"; you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-----------------------------------------------------------------------*/
+#include "resqml2_0_1/RockFluidUnitFeature.h"
+
+using namespace std;
+using namespace RESQML2_0_1_NS;
+using namespace gsoap_resqml2_0_1;
+
+const char* RockFluidUnitFeature::XML_TAG = "RockFluidUnitFeature";
+
+RockFluidUnitFeature::RockFluidUnitFeature(soap* soapContext, const string & guid, const string & title)
```

Comment:
Must add a FluidBoundaryTop and a FluidBoundaryBottom as parameters of this constructor which should be of type "FuildBoundaryFeature"

Reply:
thank you! I will had these

Reply:
The phase attribute is also mandatory and should be added to the constructor parameter list

Reply:
ok!